velox/exec/ExchangeClient.h

```diff
@@ -31,12 +31,14 @@ class ExchangeClient {
   ExchangeClient(
       std::string taskId,
       int destination,
+      int64_t maxQueuedBytes,
       memory::MemoryPool* pool,
-      int64_t maxQueuedBytes)
+      folly::Executor* executor)
       : taskId_{std::move(taskId)},
         destination_(destination),
         maxQueuedBytes_{maxQueuedBytes},
         pool_(pool),
+        executor_(executor),
```

Comment:
nit: perhaps, VELOX_CHECK_NOT_NULL(executor)

Reply:
@mbasmanova not sure if we can check not null here as we do support single-thread execution mode at least in Meta which might not provide the driver executor in query ctx. I think they might also use exchange client? Thanks!

Reply:
@mbasmanova Gluten implements a native exchange, so we are not using ExchangeClient. But we are using single-thread execution mode. @zhztheplayer Could you help confirm?

Reply:
@rui-mo Rui, thank you for clarifying.